### PR TITLE
Updated group information and image

### PR DESCRIPTION
**Pull request by meghasalunke1**
Added image and updated information

### CODE REVIEW

**Comment by meghasalunke1** [comment]
@miq-bot add_label bug

**Comment by meghasalunke1** [comment]
@miq-bot add-label oparin/yes?

**Comment by meghasalunke1** [comment]
@miq-bot assign @Fryguy

**Comment by meghasalunke1** [comment]
@miq-bot add-label petrosian/yes?

File: images/.DS_Store
Comment (Fryguy) [inline images/.DS_Store:1]: Please remove .DS_Store file as it should not be committed.

Consider using a global gitignore to ignore these files. See https://git-scm.com/docs/gitignore

**Review by Fryguy — APPROVED** [review]
Overall looks good, please just remove the one file and squash the commit.

**Comment by meghasalunke1** [comment]
@Fryguy Removed .DS_Store files from the commits.

**Comment by jrafanie** [comment]
@Fryguy I squashed and force pushed.  Please review/merge.

**Comment by Fryguy** [comment]
Backported to `petrosian` in commit e6b898590e5986f6401b0ca90b01a1791358b8a1.

```text
commit e6b898590e5986f6401b0ca90b01a1791358b8a1
Author: Jason Frey <fryguy9@gmail.com>
Date:   Tue May 2 19:30:21 2023 -0400

    Merge pull request #1730 from meghasalunke1/MS02
    
    Updated group information and image
    
    (cherry picked from commit 8e95532631a666bcd566f9123af2992876c04fae)
```

**Pull request merged**